gatekeeper/routes/routes.go

```diff
@@ -0,0 +1,24 @@
+package routes
+
+import (
+	"github.com/go-zoo/bone"
+
+	"github.com/manifoldco/torus-cli/api"
+)
+
+// NewRoutesMux returns a new *box.Mux for handling Gatekeeper requests
+func NewRoutesMux(org, team string, api *api.Client) *bone.Mux {
+	mux := bone.New()
+
+	mux.SubRoute("/machine", newMachineBootstrapRoutes(org, team, api))
+
+	return mux
+}
+
+func newMachineBootstrapRoutes(org, team string, api *api.Client) *bone.Mux {
+	mux := bone.New()
+
+	mux.PostFunc("/aws", awsBootstrapRoute(org, team, api))
+
+	return mux
+}
```

Comment:
This file is a lot of boilerplate for a single endpoint; how about just wiring it up in the Listen func? when there are new auth types added, we can rework the route declaration then.